### PR TITLE
Automation Against Humanity story

### PR DESCRIPTION
**Pull request by knorrium**

### CODE REVIEW

**Review by flynnbops — COMMENTED** [review]
Nice story. I like the conversational tone. Some of the sentences are long, and could benefit with breaking into smaller sentences.

File: felipeknorrkuhn/FELIPE_KNORR_KUHN.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Fast forward a few months, in November 2020, the Sony PlayStation 5 and the Microsoft Xbox One Series consoles launched, but getting ahold of them was, again, almost as hard as finding water in the desert. Gamers, who were still under the shelter-in-place orders, were desperate to buy their units, but something interesting and unfortunate prevented them from ordering them.
+
+Scalpers identified a good oportunity to profit with these highly anticipated videogame consoles and used specialized bots to secure multiple units before legitimate customers could have a chance to buy one. Some bots were built using popular UI automation libraries well known in the software industry. Others had direct access to APIs provided by retailers to third-party sellers, which enabled them to detect when new stock was available and allowed them to place an order before the items showed up on the storefronts.
```
Comment (LouiseWoodhams24) [inline felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: opportunity needs another p

Comment (LouiseWoodhams24) [reply felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: consider just putting November as it is just a few months later and 2020 was mentioned previously

File: felipeknorrkuhn/FELIPE_KNORR_KUHN.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Talking about videogames more specifically, during the first weeks of March 2020, the demand for a Nintendo Switch fitness game called Ring Fit Adventure was so high in countries already under shelter-in-place orders that it went out of stock globally. Getting out of the door to exercise was not going to be possible until further notice, and people who wanted a copy of the game were out of luck unless they paid two or three times over retail price for one. It took several months for additional units to become available, and only over one year after the pandemic started, some stores marked down the game's price as the interest - and the sense of necessity - faded away.
+
+Fast forward a few months, in November 2020, the Sony PlayStation 5 and the Microsoft Xbox One Series consoles launched, but getting ahold of them was, again, almost as hard as finding water in the desert. Gamers, who were still under the shelter-in-place orders, were desperate to buy their units, but something interesting and unfortunate prevented them from ordering them.
```
Comment (LouiseWoodhams24) [inline felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: ahold should be a hold

Comment (LouiseWoodhams24) [inline felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: Consider maybe "as hard as finding an oasis in a desert"

File: felipeknorrkuhn/FELIPE_KNORR_KUHN.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+The extra traffic and load generated by the bots caused massive outages. Retailers had to develop mechanisms to slow down the bots or completely prevent them from working. These new security measures included virtual queues, extra CAPTCHA challenges, and lottery systems, each with their pros and cons and possibly different ways to be defeated.
+
+This cat-and-mouse game is not new. A quick search can reveal several websites offering automation services for online stores. Being at home for so long definetly contributed to the growth and success of this kind of service. By the end of December, scalpers had [made over $28M](https://www.tomshardware.com/news/analysis-of-scalping-Q4-2020) from reselling more than 60,000 video game consoles on eBay alone with profits ranging from 150% to 350% over the original retail prices.
```
Comment (LouiseWoodhams24) [inline felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: "definetly" should be "definitely"

File: felipeknorrkuhn/FELIPE_KNORR_KUHN.md (hunk 1 above)
Comment (LouiseWoodhams24) [inline felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: specialized is the en-US version of specialised. Not sure if the content is en-US or en-GB?

**Review by LouiseWoodhams24 — COMMENTED** [review]
Great story and I felt like I learnt a bunch as well. Some of the sentences are quite long in places (I too am a long sentence human) Happy to re-review once changes have been made!

File: felipeknorrkuhn/FELIPE_KNORR_KUHN.md (hunk 2 above)
Comment (knorrium) [reply felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: Today I learned that `ahold` is Americanism - I'll change it :)

https://www.grammarly.com/blog/ahold-vs-a-hold/#:~:text=A%20hold%20means%20a%20grasp,widely%20used%20outside%20the%20U.S.

File: felipeknorrkuhn/FELIPE_KNORR_KUHN.md (hunk 1 above)
Comment (knorrium) [reply felipeknorrkuhn/FELIPE_KNORR_KUHN.md]: As we discussed on Slack, we are not normalizing to en-GB so I'll keep it as is :)

**Review by LouiseWoodhams24 — APPROVED** [review]
Fantastic!

**Pull request merged**